extensions/machine-learning-services/config.json

```diff
@@ -26,6 +26,7 @@
 	"modelManagement": {
 		"registeredModelsDatabaseName": "MlFlowDB",
 		"registeredModelsTableName": "artifacts",
+		"registeredModelsTableSchemaName": "dbo",
```

Comment:
always dbo or can be of user choice?

Reply:
it's in the config so user can change it

---
In reply to: [389164287](https://github.com/microsoft/azuredatastudio/pull/9450#discussion_r389164287) [](ancestors = 389164287)